rules/windows/process_creation/proc_creation_win_findstr_download.yml

```diff
@@ -0,0 +1,43 @@
+title: Remote File Download Via Findstr.EXE
+id: 587254ee-a24b-4335-b3cd-065c0f1f4baa
+related:
+    - id: bf6c39fc-e203-45b9-9538-05397c1b4f3f
+      type: obsoletes
+status: experimental
+description: |
+    Detects execution of findstr with specific flags and a rmeote share path. This specific sets of CLI flags would allow "findstr" to download the content of the file located on the remote share as described in the LOLBAS entry.
```

Comment:
```suggestion
    Detects execution of "findstr" with specific flags and a remote share path. This specific set of CLI flags would allow "findstr" to download the content of the file located on the remote share as described in the LOLBAS entry.
```